clusterloader2/testing/density/config.yaml

```diff
@@ -1,11 +1,17 @@
+# This test should be run on cluster with 100+ nodes!
+
 #Constants
 {{$NODES_PER_NAMESPACE := 100}}
 {{$PODS_PER_NODE := 30}}
+{{$MIN_LATENCY_PODS := 500}}
 #Variables
 {{$totalPods := MultiplyInt $PODS_PER_NODE .Nodes}}
 {{$namespaces := DivideInt (SubtractInt .Nodes 1) $NODES_PER_NAMESPACE | AddInt 1}}
 {{$replicas := DivideInt $totalPods $namespaces}}
 {{$totalPodsRounded := MultiplyInt $namespaces $replicas}}
```

Comment:
> The first thing that I would do is to understand why we really want all those Timers?

The main reasons are:
- it creates json file with all timers measurements. It can be easily parsed.
- allows to measure execution time between any two points.
- allows to connect execution time with a label. Even if we add timers in background to every phase, we still want to name these phases.

> One quick observation is that if you want to measure all phases, maybe that should just happen in the background.

I would be OK with this solution. However it's only because I know how cluster loader works.
IMHO if we want cluster loader to user friendly, we should have more explicit declaration approach instead of things running in the background.

Reply:
What I'm saying is that I don't see real benefits from having those (at least not enough benefits to justify 25% bigger config).
1. We are not tracking those times anywhere now
2. Just having a log" Phase %s started", "phase %s ended" is more than we need now.

I agree that if we want to track latency of some phase, it's useful. But we really don't care about it now.

Reply:
> We are not tracking those times anywhere now

There is a problem with Timer name - I need to revert it's name to `TestPhaseTimer` (backward compatibility). If I do this, the results will appear on perfdash.

> Just having a log" Phase %s started", "phase %s ended" is more than we need now

Just to clarify, what is an example of a text that %s will be replaced with?

Reply:
Discussed offline.
To summarize:
- let's introduce an optional "name" field to Step
- if it is defined, fire a timer for that step automatically

Reply:
Adding "name" field - #257

Reply:
Adding timer - #258

Reply:
Well - it's not timer per se, just some logs. But yes - it's definitely enough (at least for now).